src/env.ts

```diff
@@ -1,3 +1,22 @@
+import { setFailed } from '@actions/core'
 import dotenv from 'dotenv'
 
-dotenv.config()
+import type { Env } from './env.types'
+
+const prepareEnv = () => {
+  dotenv.config()
+
+  if (!process.env.GITLAB_TOKEN) {
+    setFailed('Please add the `GITLAB_TOKEN` to the changesets action')
+  }
+
+  const env = { ...process.env }
+  env.GITLAB_CI_USER_EMAIL ??= '"gitlab[bot]@users.noreply.gitlab.com"'
```

Comment:
```suggestion
  env.GITLAB_CI_USER_EMAIL ||= '"gitlab[bot]@users.noreply.gitlab.com"'
```

Email is required for committing, so empty string won't work.